oauth2/src/main/java/com/linecorp/armeria/internal/common/auth/oauth2/CaseUtil.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.internal.common.auth.oauth2;
+
+import javax.annotation.Nullable;
+
+import com.google.common.base.Ascii;
+
+import com.linecorp.armeria.common.annotation.UnstableApi;
+
+@UnstableApi
+public final class CaseUtil {
+
+  @Nullable
+  public static String firstUpperCase(@Nullable String word) {
+    if (word == null || word.isEmpty()) {
```

Comment:
global comment in this file: Please use 4 spaces for indentation.

Reply:
fixed